src/features/previewConfig.ts

```diff
@@ -47,7 +49,9 @@ export class AsciidocPreviewConfiguration {
     this.fontSize = Math.max(8, +asciidocConfig.get<number>('preview.fontSize', NaN))
     this.lineHeight = Math.max(0.6, +asciidocConfig.get<number>('preview.lineHeight', NaN))
 
-    this.styles = asciidocConfig.get<string[]>('styles', [])
+    this.styles = asciidocConfig.get<string[]>('styles', []) // REMIND: unused, we should either use it or remove it!
```

Comment:
I think we should remove it. Asciidoctor provides in-document mechanisms for this and I'm not sure there is a "global need for a different style" via a setting, particularly if it is non-functional now.

Reply:
It might be a failed attempt to replicate what the Markdown extension is doing: https://github.com/microsoft/vscode/blob/3e5c7e2c570a729e664253baceaf443b69e82da6/extensions/markdown-language-features/src/features/previewConfig.ts#L49

But the Markdown extension do have a configuration named `styles`: https://github.com/microsoft/vscode/blob/3e5c7e2c570a729e664253baceaf443b69e82da6/extensions/markdown-language-features/package.json#L275-L283

> Asciidoctor provides in-document mechanisms for this and I'm not sure there is a "global need for a different style" via a setting, particularly if it is non-functional now.

I don't think it's a "global style", I think it was just a hastily copied and pasted. The Markdown extension supports a list of stylesheets and eventually I think we should too. But one thing at a time 😉